addons/api/fnc_mapChannelFieldName.sqf

```diff
@@ -16,7 +16,9 @@
  */
 #include "script_component.hpp"
 
-params["_radioId", "_fieldName"];
+ACRE_DEPRECATED(QFUNC(mapChannelFieldName),"2.5.0","None");
```

Comment:
It's actually used in 2 places, not sure what to do... doesn't look deprecated to me?
- https://github.com/IDI-Systems/acre2/blob/f2bdfe166e162ee23e7aad143fa9d72f56e6c28c/addons/api/fnc_getPresetChannelField.sqf#L52
- https://github.com/IDI-Systems/acre2/blob/f2bdfe166e162ee23e7aad143fa9d72f56e6c28c/addons/api/fnc_setPresetChannelField.sqf#L67